app/src/main/java/com/github/kulebin/myfoursquareapp/ContextHolder.java

```diff
@@ -0,0 +1,19 @@
+package com.github.kulebin.myfoursquareapp;
+
+import android.content.Context;
+
+public enum ContextHolder {
```

Comment:
It's not clear for me what I should do. Should I change implementation and use class instead of enum? Or it is just comment for me to be aware that such approach is not reliable?